cmd/root.go

```diff
@@ -147,7 +170,11 @@ func rootCmd(o *options.Options) error {
 		return repoResult.Checks[i].Name < repoResult.Checks[j].Name
 	})
 
-	if o.Format == options.FormatDefault {
+	if len(enabledProbes) > 0 {
+		for _, probeName := range enabledProbes {
+			fmt.Fprintf(os.Stderr, "Finished probe %s\n", probeName)
+		}
```

Comment:
same comment about FormatDefault

Reply:
Done in https://github.com/ossf/scorecard/pull/3688/commits/7b40c188917c5526a174db3f1e4a2076bc15accc